ecosystem/gradle-example/build.gradle

```diff
@@ -0,0 +1,73 @@
+plugins {
+    id 'java'
+    id 'war'
+    id "com.bmuschko.cargo" version "2.3"
+}
+
+group = 'fish.payara.examples'
+version = '1.0-SNAPSHOT'
+
+description = "description of project"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+// Configure the cargo related properties.
+// Configured to -P provided options, then properties file, then environment variable, then default
```

Comment:
This won't allow specifying properties with -P and properties files as the comment says - Gradle is not as clever as Maven. That's why I used a wrapper function in my PR that checks if the properties are defined ;-) But this is an example, you don't have to make it bullet proof, just remove the comment about -P and properties file. See https://stackoverflow.com/questions/14388458/how-can-i-override-a-property-defined-in-build-gradle#33414019. I also thought it works as the comment says, but tried it and it doesn't

Reply:
Which Gradle version were you using? I tested it and it works as per the comment. Just `gradle build -PpayaraHome=test` and it should print `Payara Home: test`

Reply:
Oh, I'm sorry, I missed that you used `project.findProperty('payaraHome')` to check if the property exists. This is what I extracted into a helper method to avoid repeating the code. But your solution works, I can see it now.